labgrid/environment.py

```diff
@@ -23,6 +24,9 @@ def __attrs_post_init__(self):
             raise NoConfigFoundError(
                 "{} is not a valid yaml file".format(self.config_file)
             )
+        for user_file in self.config.get_imports():
+            importlib.import_module(user_file)
```

Comment:
Don't we need to resolve the path relative to the environment file, before passing it to importlib? We should use the same behavior as images or normal paths in the config.